docker-compose.yaml

```
@@ -16,19 +16,19 @@ services:
-#    volumes:
-#    - ./frontend:/app
-#    - ./frontend/node_modules:/app/node_modules
-#    - bun_cache:/root/.bun/install/cache
+  frontend:
+    <<: *common
+    build:
+      context: .
+      dockerfile: ./ci/dev/frontend/Dockerfile
+    command: bun --bun dev
+    volumes:
+    - ./frontend:/app
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)